nwb_conversion_tools/gui/classes/forms_basic.py

```diff
@@ -101,9 +105,20 @@ def make_forms(self):
             # String types
             if field['type'] == 'str':
                 form = QLineEdit('')
+                if field['required']:
+                    form.setPlaceholderText(field['default'])
+                else:
+                    form.setText(field['default'])
```

Comment:
I don't know the whole codebase, but these lines still seem wrong to me. If there is a default, that default should be entered into the field, not as a placeholder text, whether or not the field is required.